doc/tutorials/acrn_ootb.rst

```diff
@@ -265,14 +282,41 @@ Use the clr-installer to build the Service VM image.
      $ sudo clr-installer -c service-os-industry.yaml
 
 
-  The ``service-os-industry.img`` will be generated at current directory.
+  The ``sos-industry.img`` will be generated at current directory.
 
 .. _deploy_ootb_service_vm:
 
 Step 3: Deploy the Service VM image
 ===================================
 
 #. Prepare a U disk with at least 8GB memory.
+#. Format the U disk
```

Comment:
```suggestion
#. Prepare a U disk with at least 8GB memory. Begin by formatting the U disk:
```

Reply:
The first step is not really a step, is it? It does not contain any instructions. Perhaps we should combine #1 and #2 as shown above.

Reply:
Fixed, thanks a lot.